yarn-project/circuits.js/src/contract/unconstrained_function_membership_proof.test.ts

```diff
@@ -40,6 +40,8 @@ describe('unconstrained_function_membership_proof', () => {
     expect(artifact.functions.filter(isUnconstrained).length).toBe(1);
 
     const unconstrainedFunction = unconstrainedFns[0];
+    const selector = FunctionSelector.fromNameAndParameters(unconstrainedFunction);
```

Comment:
This shadows the `selector` variable in the test - I'd pick a different name, or pass this value directly to the fn call

Reply:
(also why do we need to recompute this if we're already doing it in the `beforeEach` block?)